src/components/table/Table.test.tsx

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+import { fireEvent, render, screen } from '@testing-library/react';
+import { columns } from '../../utils/tableColumns';
+import { mockRows } from '../../utils/MockData';
+import { Table } from './Table';
+
+describe('Components Render Correctly', () => {
+  beforeAll(() => {
+    render(<Table rows={mockRows} columns={columns} />);
+  });
+  it('Renders Table without crashing', () => {
+    console.log(screen.debug());
```

Comment:
remover este console log. testea que la informacion si se renderice

Reply:
Deleted it and then changed it to resolve next comment.